src/data-engine.js

```diff
@@ -142,8 +142,8 @@ class DataEngine {
     let directParentList = this.getDirectListParentOfItem(topParent, item)
 
     if (!directParentList) {
-      // so we need to create the direct parent UL and append it to the direct parent LI
-      directParentList = this.createItemElement({ id: parent }, 'ul')
+      // so we need to create the direct parent OL and append it to the direct parent LI
```

Comment:
personal preference I would remove the "so" from this

Reply:
Done! 👍🏼